Assets/Packages/com.studio23.ss2.roomloadingsystem/Runtime/Core/RoomInstance.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Studio23.SS2.RoomLoadingSystem.Core
+{
+    public class RoomInstance:MonoBehaviour
+    {
+        [FormerlySerializedAs("defaultPlayerSpawnPoint")] 
+        public Transform _defaultPlayerSpawnPoint;
+        [FormerlySerializedAs("Room")] 
+        public RoomData _room;
+        public bool DoesRoomPosMatch() => _room.WorldPosition == transform.position;
+
+        private void Awake()
+        {
+            if (!DoesRoomPosMatch())
+            {
+                UnityEngine.Debug.Log($"Room {_room} worldpos {_room.WorldPosition} doesn't match roominstance worldPos {transform.position}");
```

Comment:
It probably should be a log Error instead of just log. People will tend to ignore. Or we should use color coding.

Reply:
Quite sure it probably already exists. And as for define symbols, you can just turn logging off from the settings.

Reply:
Moving to logwarning for now.